Sources/TuistKit/Commands/Cache/CacheWarmCommand.swift

```diff
@@ -23,7 +23,10 @@ struct CacheWarmCommand: ParsableCommand {
     )
     var xcframeworks: Bool = false
 
+    @Argument(help: "A list of targets to cache. Those and their dependant targets will be cached.")
+    var sources: [String] = []
```

Comment:
maybe targets would be more comprehensible? or what is the reasoning for sources? I think `tuist cache warm --targets FrameworkA` reads better. Or, maybe better yet, it could `tuist cache warm --modules FrameworkA`. What do you think?

Reply:
Okay reading better the help hint from the other command: `A list of targets in which you'd like to focus. Those and their dependant targets will be generated as sources.` it may make sense to have a different one for this one. Wouldn't `targets` be better over `modules` as we don't mention in the help hint?

Reply:
yeah, `targets` sounds better to me - the use case is a little bit different than for `tuist focus`, so I would not worry about strictly sticking to that naming